website/webpack/markdown-loader.js

```diff
@@ -14,6 +14,7 @@ const cheerio = require("cheerio");
 const path = require("path");
 const slug = require("slug");
 const resolve = require("resolve-pathname");
+const loaderUtils = require("loader-utils");
```

Comment:
Could you add `loader-utils` as a `devDependency`? It is already installed because it is a dependency of `webpack`, but since we are referencing it ourselves, I'd prefer for it to be listed in the package's `package.json`.

Reply:
Sure thing!